content/en/about-types.md

```diff
@@ -0,0 +1,261 @@
+ </td>
+ <td><p>
+  JSDoc supports Closure Compiler's syntax for defining array and object types.
+  <p>
+  <p>
+  You can also indicate an array by appending <code>[]</code> to the type that is contained in the
+  array. For example, the expression <code>string[]</code> indicates an array of strings.
+  </p>
+  <p>
+  For objects that have a known set of properties, you can use Closure Compiler's syntax for
+  documenting record types. You can document each property individually, which enables you to
+  provide more detailed information about each property.
+  </p>
+ </td>
+</tr>
+
+<tr>
+ <td>
+  Nullable type
+ </td>
+ <td>
+{% example "A number or null." %}
+<pre class="prettyprint"><code>{?number}
+</code></pre>
+{% endexample %}
+ </td>
+ <td>
+  <p>
+  This indicates that the type is either the specified type, or <code>null</code>.
+  </p>
+ </td>
+</tr>
+
+<tr>
+ <td>
+  Non-nullable type
+ </td>
+ <td>
+{% example "A number, but never null." %}
+<pre class="prettyprint"><code>{!number}
+</code></pre>
+{% endexample %}
+ </td>
+ <td>
+  <p>
+   Indicates that the value is of the specified type, but cannot be <code>null</code>.
+  </p>
+ </td>
+</tr>
+
+<tr>
+ <td>
+  Variable number of that type
+ </td>
+ <td>
+{% example "This function accepts a variable number of numeric parameters." %}
+<pre class="prettyprint"><code>@param {...number} num
+</code></pre>
+{% endexample %}
+ </td>
+ <td>
+  <p>
+  Indicates that the function accepts a variable number of parameters, and specifies a type for the
+  parameters. For example:
+  </p>
+{% example %}
+<pre class="prettyprint"><code>/&ast;&ast;
+ &ast; Returns the sum of all numbers passed to the function.
+ &ast; @param {...number} num A positive or negative number
+ &ast;/
+function sum(num) {
+    var i=0, n=arguments.length, t=0;
+    for (; i&lt;n; i++) {
+        t += arguments[i];
+    }
+    return t;
+}
+</code></pre>
+{% endexample %}
+ </td>
+</tr>
+
+<tr>
+ <td>
+  Optional parameter
+ </td>
+ <td>
+{% example "An optional parameter named foo." %}
+<pre class="prettyprint"><code>@param {number} [foo]
+// or:
+@param {number=} foo
+</code></pre>
+{% endexample %}
+
+{% example "An optional parameter foo with default value 1." %}
+<pre class="prettyprint"><code>@param {number} [foo=1]
+</code></pre>
+{% endexample %}
+ </td>
+ <td>
+  <p>
+  Indicates that the parameter is optional. When using JSDoc's syntax for optional parameters, you
+  can also indicate the value that will be used if a parameter is omitted.
+  </p>
+ </td>
+</tr>
+
+<tr>
+ <td>
+  Callbacks
+ </td>
+ <td>
+{% example %}
+<pre class="prettyprint"><code>/&ast;&ast;
+ &ast; @callback myCallback
+ &ast; @param {number} x - ...
+ &ast;/
+
+/&ast;&ast; @type {myCallback} &ast;/
+var cb;
+</code></pre>
+{% endexample %}
+ </td>
+ <td>
+  <p>
+  Document a callback using the <a href="tags-callback.html">@callback</a> tag. The syntax is
+  identical to the @typedef tag, except that a callback's type is always "function."
```

Comment:
enclose tag name in `<code>`, and change `"function."` to `<code>function</code>.`

Reply:
Done